app/service/file_svc.py

```diff
@@ -78,12 +80,14 @@ def __init__(self, plugins, exfil_dir):
         return path
 
     async def _compile(self, name, platform):
+        if platform.lower() not in ["windows", "linux", "darwin"]:
```

Comment:
instead of hard-coding the platforms here (as we have plans to expand into other areas in the near future) let's - in the __init__ of the file_svc - collect the unique platforms from the database.